### PR TITLE
Airship2 GA Announcement

### PR DESCRIPTION
**Pull request by alexander-hughes**
Signed-off-by: Alexander Hughes <Alexander.Hughes@pm.me>

### CODE REVIEW

File: src/pages/blog/airship2-released.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+<br>
+
+* **Homogeneous -> Equivalency** <br> Regarding our homogenous goal, we deliver a level of uniformity in terms of how we lifecycle the clouds themselves. We also provide a mechanism to ensure these clouds' configuration has a homogeneous feel and adheres to our internal enterprise requirements -- things like standardizing Admission Controllers, Webhooks, Network Security Policies, Namespaces, and more. The advantage airshipctl provides is it works with anything presenting a Kubernetes API. Because of its integration with Kustomize and our ability to layer documents in various ways, from inheriting global values to bifurcating on the type and finally site-specific customizations -- it provides an ideal way to deliver a homogeneous configuration across a variety of cloud deployments and infrastructure backends.
```
Comment (intlabs) [inline src/pages/blog/airship2-released.md:39]: `our` is not appropriate here.

**Review by intlabs — CHANGES_REQUESTED** [review]
I'm making some changes to the google doc that this is sourced from - please can we incorporate them before merge

**Comment by alexander-hughes** [comment]
> 
> 
> I'm making some changes to the google doc that this is sourced from - please can we incorporate them before merge

Certainly.  Please send me the link, or updated word doc when ready. These changes were sourced from a word doc provided by Rodolfo late last week.

File: src/pages/blog/airship2-released.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+<br>
+
+* **Multi Use-Case** <br> We intend to use Airshipctl to drive cloud installations and lifecycle management of environments running lots of different workloads, from CI/CD, to CNFs, to VNFs, to a hybrid between both. We've made airshipctl as simple to use as possible for new community users, folks who need to stand up four servers in a lab but lend itself to those same users if they want to take it into production with 100 servers or 100's of clouds. It is also straightforward for operators to leverage workload definitions from the Airship Treasuremap project and seamlessly integrate their own workloads.
+
+<br>
+
+* **Cloud Agnostic -> Any Cloud** <br> With Airship 2, we have fully embraced the cluster-api and have integrated it closely into airshipctl. We support a variety of infrastructure backends and operating system choices, from public clouds (Azure, Google), to VMs (Openstack), to bare metal (Meta3). We have unified how we stand up, configure, and lifecycle cloud infrastructure across these various providers with a single tool.
```
Comment (jtwill98) [inline src/pages/blog/airship2-released.md:35]: Meta3 (sp, typo)  should be Metal3

**Review by jtwill98 — CHANGES_REQUESTED** [review]
One typo issue other wise lgtm.

File: src/pages/blog/airship2-released.md (hunk 2 above)
Comment (alexander-hughes) [reply src/pages/blog/airship2-released.md]: Thanks, resolved in latest revision.

**Pull request merged**